packages/ui/src/components/TextInputV2/index.tsx

```diff
@@ -133,9 +132,8 @@ type TextInputProps = {
   onChange?: (newValue: string) => void
   prefix?: ReactNode
   size?: TextInputSize
-  success?: string
+  success?: string | boolean
```

Comment:
new spec ? not checking the type could lead to not wanted TextComponent as helper

Reply:
Yep new specs, I will add a verification for the text part